lib/google_maps_service/apis/roads.rb

```diff
@@ -63,8 +63,9 @@ def snap_to_roads(path, interpolate: false)
     #         by the snap_to_roads function. You can pass up to 100 Place IDs.
     #
     # @return [Array] Array of speed limits.
-    def speed_limits(place_ids)
+    def speed_limits(place_ids, speed_unit='KPH')
       params = GoogleMapsService::Convert.as_list(place_ids).map { |place_id| ['placeId', place_id] }
+      params << ['units', speed_unit] if speed_unit == 'MPH'
```

Comment:
In this situation, params is an array. `[['key1','value1'],['key2','value2']]`. I went ahead and made the conversion from the multi-dimensional array to a hash. Would you rather it converted to a hash for readability or to use the << or push methods to add the the array?

`snapped_speed_limits` params are in hash form so this isn't a concern there.

Fyi, when i converted the params to a hash and added the units as a symbol, it created problems. Therefore I went with `params['units']` instead inside `speed_limits` method.